### PR TITLE
make powered by esri omnipresent

### LINKED ISSUE #838: 'Powered by Esri' should always be present in the attribution
currently, we only [tweak](https://github.com/Esri/esri-leaflet/blob/master/src/Layers/BasemapLayer.js#L205) Leaflet's attribution prefix when an Esri basemap is added to the map.

since this attribution should be displayed when **any** content from ArcGIS Online is being drawn, the easiest thing to do is just call `setPrefix()` as soon as this plugin is loaded instead of waiting for a basemap to be added.

### PR DESCRIPTION
**Pull request by jgravois**
resolves #838. i :heart: ES6.

### CODE REVIEW

File: src/Layers/FeatureLayer/FeatureManager.js [hunk 1]
```diff
@@ -64,6 +64,9 @@ export var FeatureManager = VirtualGrid.extend({
    */
 
   onAdd: function (map) {
+    // include 'Powered by Esri' in map attribution
+    Util.setEsriAttribution(map);
```
Comment (ta2edchimp) [inline src/Layers/FeatureLayer/FeatureManager.js]: Shouldn't this line read `setEsriAttribution(map)` only, as you're explicitly importing `setEsriAttribution`?

Comment (jgravois) [inline src/Layers/FeatureLayer/FeatureManager.js]: yes it should. thx! :wink:

**Comment by ta2edchimp** [comment]
Do you have any idea why the travis build fails?
I fetched your branch, pushed it to my fork and [triggered a build which completed successfully](https://travis-ci.org/ta2edchimp/esri-leaflet/builds/158221498).  
Only reason I can think of would be a corrupted build cache or sth like that (the only related difference I spotted between this repo's `.travis.yml` and [those I'm used to](https://github.com/ta2edchimp/opt-cli/blob/master/.travis.yml#L15-L16), is that I run `npm prune` as `before_script`, as the `node_modules` directory is added to travis' cache).

**Comment by jgravois** [comment]
> Do you have any idea why the travis build fails?

i really wish i did. 

i've tried a couple times lately, and manually blowing away the cache doesn't typically resolve the problem.  to make things even more strange, i was able to dependably *break* the build both locally and on travis by pointing at Node v6.x (see #815) and see green when i reverted to v4.x.  but now i'm seeing consistently reproducible errors (even locally) when targeting v4.5.0.

i'll try injecting `npm prune` and see if it helps.  either way, i appreciate the suggestion!

**Comment by ta2edchimp** [comment]
I hope it helps, or at least somehow points into the right direction.
(I originally stumbled over this PR when I was looking into the node6 branch.)

**Comment by ta2edchimp** [comment]
Another thing that just came into my mind:
We experienced serious issues using npm@2.x (for example, over at [`ghooks`](https://github.com/gtramontina/ghooks/issues/71)), as it does some really strange things during npm's post/preinstall hooks and got used to add
```
before_install:
  - npm i -g npm@^3.0.0
```
to our travis configs. Idk if this is related, but I would give it a try (as the failing builds seem somehow random here, too).

**Comment by jgravois** [comment]
still seeing flaky behavior, but either way i appreciate the help.
makes me even more glad to learn there's a flag to turn off email notifications!

**Comment by ta2edchimp** [comment]
Maybe contacting Travis support can bring some insight. They usually respond fast ...

**Comment by ta2edchimp** [comment]
Could you change karma's log level to something more verbose?
I'm irritated not only about the failing tests (that look somehow random), but also by the number of tests run (in the Travis builds, not even half of the tests are run — reminds me of PhantomJS' memory or more general issues on Linux from some time ago, although those errors should be printed out, given even the current log level).

**Comment by jgravois** [comment]
after pushing a commit that sets the logging level to `verbose`, the build is green again.  the next time i see irregular behavior i'll take your advice and contact Travis Support.

**Comment by ta2edchimp** [comment]
At least, it's green! :tada: 
I'm curious if this makes the other PR work, too.

**Comment by jgravois** [comment]
> At least, it's green! 🎉 

absolutely.  i really appreciate your persistence because we've been seeing stuff in Travis that i don't completely understand for a *long* time. 

> I'm curious if this makes the other PR work, too.

i'll rebase it on top of this branch so we can see 😬

**Comment by jgravois** [comment]
merging.  

much thanks to @ta2edchimp for his suggestions related to getting travis to behave.  i'll observe the behavior a bit and let you know how we get on soon.

**Pull request merged**